packages/web/src/components/gcds-fieldset/stories/gcds-fieldset.stories.tsx

```diff
@@ -89,12 +51,8 @@ const Template = args =>
 <gcds-fieldset
   fieldset-id="${args.fieldsetId}"
   legend="${args.legend}"
+  size="${args.size}"
```

Comment:
We will have to update this to the new `legendSize` attribute name

Reply:
Oops, you are 100% correct - thank you!